packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -342,6 +396,13 @@ class Chat extends Widget<ChatProperties> {
     );
   }
 
+  _createMessageEditCanceledAction(): void {
```

Comment:
You pass only message argument to the Chat back, so the common callback will sufficient here